### PR TITLE
Add iceberg connector doc

### PR DESCRIPTION
**Pull request by ChunxuTang**
Add iceberg connector doc

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by beinan — APPROVED** [review]
Great doc! thank you Chunxu!  
 @zhenxiao do you also want to take a look!

**Review by zhenxiao — CHANGES_REQUESTED** [review]
thank you, @ChunxuTang 
one minor comment

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Configuration Properties
+------------------------
+
+The following configuration properties are available:
+
+================================== ===================================================
+Property Name                      Description
+================================== ===================================================
+``hive.metastore.uri``             The URI(s) of the Hive metastore.
```
[the remark this answers is not in the document]
Comment (ChunxuTang) [reply presto-docs/src/main/sphinx/connector/iceberg.rst]: Nice catch! Sure, I added descriptions of these two properties in the section `Iceberg Configuration Properties`.

Yeah, the metastore uri is a must. A user needs to provide this property to use the iceberg connector~

**Review by zhenxiao — APPROVED** [review]
nice.
@ChunxuTang could you please squash the 2 commits into 1?
BTW, in our code, do we have config requirements for hive metastore url?

**Comment by ChunxuTang** [comment]
> nice.
> @ChunxuTang could you please squash the 2 commits into 1?
> BTW, in our code, do we have config requirements for hive metastore url?

@zhenxiao Thanks for your suggestion! Sure, I've squashed the 2 commits into 1.

For the hive metastore uri, it's not extracted in the `IcebergConfig`. But actually, the iceberg connector uses the HiveMetastoreModule with `StaticMetastoreConfig` bound. That config extracts the hive metastore uri.

**Pull request merged**